modules/lang-groovy/src/test/java/org/elasticsearch/messy/tests/SearchFieldsTests.java

```diff
@@ -367,6 +367,28 @@ public void testScriptFieldUsingSource() throws Exception {
         assertThat(((Map<?, ?>) sObj2Arr3.get(0)).get("arr3_field1").toString(), equalTo("arr3_value1"));
     }
 
+    public void testScriptFieldsForNullReturn() throws Exception {
+        createIndex("test");
```

Comment:
You can skip these two and just create the index. It ought to just automatically create.